### PR TITLE
docs: align Github templates

### PR DESCRIPTION
**Pull request by escapedcat**
Adjusted all templates:

- Use headlines instead of just bold to create automatic margin
- All entries have _descriptions_ to indicate what this means and that these can be removed
- Add further info

What do you think? Anything else we should add or improve?

### CODE REVIEW

**Comment by escapedcat** [comment]
Not exactly sure why this fails?
image
Because this is not changing any code?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+### Type of change (Remove other not matching type)
 
-#### Describe the changes you have made in this PR -
+- `fix`: Bug fix (non-breaking change which fixes an issue)
+- `feat`: New feature (non-breaking change which adds functionality)
+- `feat!`: Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- `docs`: Documentation update
 
-#### Screenshots of the changes (If any) -
+### Describe the changes you have made in this PR
```
Comment (bumi) [inline .github/pull_request_template.md:12]: what do you think about moving this higher - above type of change? 
I found this more relevant so far

**Review by bumi — COMMENTED** [review]
👍

**Comment by bumi** [comment]
@escapedcat I hade played around with action permissions, that error was may fault.

File: .github/pull_request_template.md (hunk 1 above)
Comment (bumi) [reply .github/pull_request_template.md:12]: 👍 means we move the "description" up? :)

Comment (escapedcat) [reply .github/pull_request_template.md:12]: Haha , wait. So current order is:
- Link issue
- Type of change
- Description
- Screenshot
- Tested
- Checklist

Ist that ok?

Comment (bumi) [reply .github/pull_request_template.md:12]: I thought about: 


1. Description
2. Link issue
3. Type of change
4. Screenshot
5. Tested
6. Checklist

Comment (escapedcat) [reply .github/pull_request_template.md]: Done

**Pull request merged**